tests/e2e/002-walletFeatures.spec.ts

```diff
@@ -39,7 +39,7 @@ test.describe("Wallet features", () => {
       extensionId,
     });
 
-    await findByText($optionsdocument, "Your ⚡️ Websites");
+    await findByText($optionsdocument, "Your Alby wallet is ready");
```

Comment:
Shouldn't `Your ⚡ Websites` be on this site, too?

Reply:
Will be changed with this: #1905